website/content/docs/installation.md

```diff
@@ -39,20 +39,20 @@ Go to [Next steps]({{< ref "#next-steps" >}}) to configure Bloop and start using
 
 ### Instructions for other platforms
 
-We do not publish native packages for other platforms at the moment, but you can use our
-Python installation script to install Bloop on your machine. The installation script has been
-reported to work on major Linux distributions, Mac OS and Windows.
-
-The installation script can be downloaded from [the Releases of Bloop]({{< githubrepo
->}}/releases).
-Run it to install Bloop in your current working directory:
+Install bloop in other platforms (Windows, Unix, \*bsd) via our python script:
 
 ```sh
-$ ./install.py
+$ curl -L https://github.com/scalacenter/bloop/releases/download/v1.0.0-M5/install.py | python2
 ```
 
-You can then move the generated binaries somewhere else on your disk. Optionally, you can also pass
-a destination where the installation script should put the binaries:
+<span class="label warning">Note</span> that you may need to update the version
+in the URL. Check the last released version in [Bloop GitHub releases]({{<
+githubrepo >}}/releases).
+
+#### Installing in a different repository
+
+By default, the script installs bloop binaries in a `.bloop` directory in your
+`$HOME` folder. You can change that with the `--dest` option:
```

Comment:
I think that the default is to install in the current working directory, no?

Reply:
`BLOOP_DEFAULT_INSTALLATION_TARGET = join(expanduser("~"), ".bloop")`

Reply:
😝 

https://github.com/scalacenter/bloop/blob/d44ecafd86414aa8a02462e91cc607ae9567cba2/bin/install.py#L26-L30

`BLOOP_DEFAULT_INSTALLATION_TARGET` is not used anywhere. What do you prefer we do?